docs/guides/hybrid-graphics.md

```diff
@@ -43,7 +43,7 @@ If you experience system freezes, then the laptop's fans becoming loud, before t
 
 2.  Set up apple-set-os-loader to make Apple's firmware show the iGPU so apple-gmux will be able to switch to it:
 
-    1.  Compile apple-set-os loader. These instructions assume you have `gnu-efi` installed, and mount your EFI partition on `/boot/efi`. If you mount the EFI partition somewhere else or use refind, you will need to replace /boot/efi with the mount point of the partition in which your bootloader is installed.
+    1.  Compile apple-set-os loader. These instructions assume you have `gnu-efi` and `build-essential` installed, and mount your EFI partition on `/boot/efi`. If you mount the EFI partition somewhere else or use refind, you will need to replace /boot/efi with the mount point of the partition in which your bootloader is installed.
```

Comment:
Build essential is an Ubuntu package. As far as I know, similar packages have other names in other distros. Can we confirm the same from somewhere?

Reply:
Yes, I thought about it that could be very specific for Ubuntu. Just try to avoid people asking basic questions in forum because they are missing basic compiling stuff.

Reply:
In found arch Linux, would be
sudo pacman -Sy base-devel

Reply:
In fedora its `dnf group install "C Development Tools and Libraries"

Reply:
sudo pacman -S gnu-efi
and sudo apt install gnu-efi

how is it in fedora?

Reply:
`dnf install gnu-efi`

Reply:
is it worth it to add an example to each of them
Ubuntu
`sudo apt install gnu-efi build-essential`
Archlinux
`sudo pacman -Sy base-devel`
`sudo pacman -S gnu-efi`
Fefora
`dnf group install "C Development Tools and Libraries"`
`sudo dnf install gnu-efi`

Reply:
Yes, these command cover all supported distros except Nix and Gentoo, so it will help a lot of users. The commands for ubuntu also apply to debian, and the arch commands are the same for eOS

Reply:
perfect. Ill  add them somehow.

Reply:
done